Dockerfile-runtime

```diff
@@ -37,10 +37,12 @@ RUN bash -c 'if [ `arch` = "ppc64le" ]; then yum install -y git boost-devel cmak
 
 RUN mkdir -p /opt/h2oai/h2o4gpu
 
+ARG python_version=3.7
```

Comment:
here too, this is overall default now, but probably want it to be controllable which python version to build.